projects/drivers/clk_axi_clkgen/clk_axi_clkgen.c

```diff
@@ -215,19 +262,68 @@ static uint32_t axi_clkgen_lookup_lock(uint32_t m)
 	return 0x1f1f00fa;
 }
 
+/**
+ * @brief axi_clkgen_setup_ranges
+ */
+static void axi_clkgen_setup_ranges(struct axi_clkgen *axi_clkgen,
+				    uint32_t *fpfd_min, uint32_t *fpfd_max,
+				    uint32_t *fvco_min, uint32_t *fvco_max)
+{
+	unsigned int reg_value;
```

Comment:
Should we have only "unsigned int" or "uint32_t" types in this file?

Reply:
I think consistency should be kept. I will change with uint32_t